### PR TITLE
 	fixed issue #1550 regarding "maintainer updates box"

### PR DESCRIPTION
**Pull request by shriyanka**

### CODE REVIEW

**Comment by willingc** [comment]
@shriyanka Thank you for submitting the pull request :sunny:

Let me make sure and clarify what I am looking at in the Pull Request:
* Documentation for #1338 
* Maintainer box UI fix for #1550

What's great is that Travis build passed which means the changes have passed all existing tests. Now, it's my turn to take a look at your actual changes.

**Comment by shriyanka** [comment]
Hi,
Update add_user_project.rst referenced above consists documentation. link https://github.com/shriyanka/oh-mainline/blob/master/docs/tutorials/add_user_project.rst
Sorry for inconvenience.
Thanks

File: docs/tutorials/add_user_project.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+- Once you are done, click on Save and Publish.
+
+Your Project will be published and you can view it by clicking on your profile.
+
+In your profile you will get the list of all your saved projects. 
+
+You can add/edit or delete a project by clicking on the add/edit link present in the right side of Projects section.
+
+You can also add a bug tracker by clicking on Add project's bugs. Find the procedure for same on our wiki page or in OpenHatch’s Documentation.
+
```
Comment (willingc) [inline docs/tutorials/add_user_project.rst]: It's good practice to remove unneeded blank lines :)

File: docs/tutorials/add_user_project.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
+========================================
+Documentation: Adding a User and Project
+========================================
+(Need Feedback)
```
Comment (willingc) [inline docs/tutorials/add_user_project.rst]: Should be removed now that the suggested documentation is in a pull request.

File: docs/tutorials/add_user_project.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+Adding a User
+=============
+
+OpenHatch community welcomes its users to initiate an active participation in various activities such as contributing to a project, Finding a bug and Adding your own Project.
```
Comment (willingc) [inline docs/tutorials/add_user_project.rst]: It's good style to keep the line length less than 79 characters. It helps future readers of your code to see it on a screen without the lines wrapping which helps in developer maintainability of the code. Editors may be set to remind you that the line is exceeding 79 characters.

File: mysite/profile/templates/profile/importer.html [hunk 4]
```diff
@@ -110,14 +110,31 @@ <h4 id='projects_heading'>
...
+            if(!jactive2){
+                jactive2=true;
+            $.jGrowl("<strong style='color: yellow;'>Maintainer updates</strong><p>Check this box to receive updates geared towards project maintainers, including: <ul style='margin-left: 2em;' class='raquo_bullets'> <li>new contributors</li> <li>people who click the 'I want to help' button</li> </ul></p>", {'life': 14000,
+                'close':function(){
+                    jactive2=false;
+                }});
             return false;
+        }
         });
+
```
Comment (willingc) [inline mysite/profile/templates/profile/importer.html]: Perhaps remove some of the blank lines.

**Comment by shriyanka** [comment]
Hi,
@willingc Thankyou for the feedback. I will surly keep each point in mind here onwards.
So my task is : remove the (need feedback)
Remove extra lines from both files.
And after that I create a pull request once again of the edited files?
Thanks.

**Comment by willingc** [comment]
@shriyanka I've had a chance to do a first pass review of the pull request and made a few comments on the files.

I think the best next step for this PR (pull request) is to use git to split the pull request into two separate PRs (one for the documentation and one for the other change). Although this may seem like more work initially, the benefits to you will be additional knowledge of git branches, git rebase, and working with git history.

A contributor recently created this guide to help with git and openhatch: http://openhatch.readthedocs.org/en/latest/advanced/working_with_git.html

It would be helpful to pair with me or someone else to walk through the process together of the particular git commands to split the pull request and how to use branches and rebase when creating a new feature or changing existing code.

**Comment by shriyanka** [comment]
@willingc  actually I am new to git so I am facing some problem here of understanding and doing the task.
So basically I need to do this:
1) edit both the files in my repo.
2) create two different branch having each of the edited files.
3) create a pull request from the branches.

The new concepts are bit confusing. Sorry for asking such questions.

**Comment by willingc** [comment]
@shriyanka Please do not apologize for asking questions. We encourage questions at OpenHatch. It's crucial to learning to ask questions.

I'm at a conference today so my apologies for brief responses. I will try to post a detailed response later today.

@shriyanka Keep up the good work on your journey to learn git. :sunflower: 

cc/ @ehashman @eeshangarg  If either of you have time to do this before I get to it I would be most grateful.

**Comment by willingc** [comment]
@shriyanka Quick question: are you using git from the command line or the visual application?

# (For command line) Let's start by checking your remote repositories that are set up.

From the oh-mainline directory on your system, `git remote -v` will give you information about which remote repositories you have configured.
screen shot 2015-03-11 at 08 49 51

There are two remote repositories set up: 'origin' (which is stored on my github account) and 'upstream' (which is stored in the openhatch github account).

If you type `git remote -v`, you will see which remotes are set up for you. My guess is that you will only see 'origin'.

What will make things helpful for you is to set up a remote that maps to the oh-mainline repository, and I recommend calling it 'upstream'.

# To create the 'upstream' remote
Type on the command line in the oh-mainline directory on your system 
    `git remote add upstream https://github.com/openhatch/oh-mainline.git`

You can now use `git remote -v` and you should see both 'origin' and 'upstream' as above.

@shriyanka Please leave me a note here. And I will post a next step after your remotes are configured :-)

**Comment by shriyanka** [comment]
@willingc I have successfully configured my remote with origin and upstream as you explained.It helped a lot :) Whats the next task/step that I should follow ?
@willingc  I had submitted some updated commits correcting the issues that you mentioned. Please do have a look in that and inform me if has some issues to be fixed. I learn a lot that way :)

**Comment by shriyanka** [comment]
Hello,

It has been few days since my last contribution as I was off track but now
once again I am on board. I know you all will be very busy but can you
suggest what could I do next for the oh-mainline.? And yes I had configured
the repositories please guide me to next part.I tried it for myself and
ended up with lot of git commands and some deleted repositories. :P

Regards,
Kajal Gupta

**Comment by willingc** [comment]
@shriyanka Thanks for continuing to follow up on this pull request. I'm glad that the git learning is progressing. Probably the next best thing to learn in git is how to do an interactive rebase. This allows you to manipulate the commit history to reorder and select which commits to submit to a PR. Try to search the existing OpenHatch docs for git rebase. Another contributor added some help tips recently on it. Feel free to ask questions about workflow and how to rebase this PR :)

File: docs/tutorials/add_user_project.rst [hunk 5]
```diff
@@ -0,0 +1,62 @@
+========================================
+Documentation: Adding a User and Project
+========================================
+
+Adding a User
+=============
+
+OpenHatch community welcomes its users to initiate an active participation in various 
+activities such as contributing to a project, Finding a bug and Adding your own Project.
```
Comment (onceuponatimeforever) [inline docs/tutorials/add_user_project.rst]: Please un-capitalize the following:
`Finding` --> `finding`
`Adding` --> `adding`
`Project` --> project

File: docs/tutorials/add_user_project.rst [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+In order to do so you have to add yourself as a user.
+The basic steps that you will require to add yourself as a User are:
```
Comment (onceuponatimeforever) [inline docs/tutorials/add_user_project.rst]: `User` --> `user`

File: docs/tutorials/add_user_project.rst [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+- You will be directed to /editor/ page of your profile.
+  Click on Add a project in the Manage projects section.
+- Once you click it a form will be displayed. Fill it as follows:
+
+
+  Type a project name: A suitable name for your project.
+  
+  Describe this project: A short description about your project.
+  
+  Check the receiver maintainer updates Box if you want to: Receive updates
```
Comment (onceuponatimeforever) [inline docs/tutorials/add_user_project.rst]: `Check the maintain updates box to receive updates`

**Comment by ehashman** [comment]
Susan's notes revived this pull request; I didn't see @willingc's ping before. Great work here!

The 10 commits on this pull request are not in a mergeable state, but I would be happy to pair with you, @shriyanka, in order to fix them up! Or if that's not something you have time for, I'd also be happy to go ahead and rebase your pull request for you to ensure your contributions here get merged :)

**Comment by shriyanka** [comment]
@onceuponatimeforever  Thank you for reviewing this PR. I will make the changes as suggested by you within 24 hrs and submit a new one :) 
@ehashman  yes the PR are in a messed up state and I would be happy to work on them with you :) How would you like me to proceed ?
I have an issue every time I make any changes in the doc it directly creates a PR and it gets messed
up again. How do I correct that ?

**Comment by shriyanka** [comment]
@willingc I will surely study how to rebase as I see there exists an urgent requirement for it right here :)

**Comment by ehashman** [comment]
Hi @shriyanka, I'm not sure why a new PR would be created every time you make a doc change. Can you give some more information about the software you are using to make your changes in git?

**Comment by shriyanka** [comment]
I am not sure why did that happen. last time when @willingc asked me to
make first changes i made the edit in the doc on github site iteslf and
once i saved the changes an automatic submission of pull request was done.

**Comment by ehashman** [comment]
Ah, @shriyanka, I understand. I've never made a pull request editing things directly on GitHub before so I've not encountered this kind of behaviour. Since you're not making the git submissions yourself, what I'd like to do is fix all the git stuff for you and I'll submit a pull request on your behalf. Let me look into that today.

**Comment by ehashman** [comment]
Closing, as this is superseded by #1620.

**Pull request closed without merging**